tests/src/FunctionalJavascript/MembershipSubmissionTest.php

```diff
@@ -133,6 +133,31 @@ public function testSubmitWebform() {
     $this->getSession()->getPage()->pressButton('Save Settings');
     $this->assertSession()->pageTextContains('Saved CiviCRM settings');
 
+    // Create two memberships with the same status with the first membership
+    // having an end date after the second membership's end date.
+    \Drupal::service('webform_civicrm.utils')->wf_civicrm_api('membership', 'create', [
```

Comment:
@puresyntax71 Can we use `$this->utils` instead of `\Drupal::service('webform_civicrm.utils')`?

Reply:
I have updated the test.